spec/factories/notification.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :notification do
+    notification_type { FactoryGirl.create(:notification_type) }
```

Comment:
Is this block needed? I thought FactoryGirl's DSL was to have associations declared with just the name?